src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecPlanGraph/DataReaderNodeBuilder.cs

```diff
@@ -37,7 +37,7 @@ public DataReaderNodeBuilder()
         /// </summary>
         /// <param name="dataSource">Data Source.</param>
         /// <returns>An array of AnalysisServices Graph objects.</returns>
-        public ShowPlanGraph[] Execute(object dataSource)
+        public ExecutionPlanGraph[] Execute(object dataSource)
```

Comment:
In the future I'd suggest that you do cleanup stuff like this in separate PRs - this is adding a ton of cruft to the PR and makes it harder to properly review.